app/controllers/concerns/wizard_controller.rb

```diff
@@ -0,0 +1,31 @@
+module WizardController
+  extend ActiveSupport::Concern
+
+  included do
+    helper_method :current_step_path, :back_link_path
```

Comment:
Let's see if we can include `:step_path` here, too. That would tidy up the individual controllers even more.

```suggestion
    helper_method :step_path, :current_step_path, :back_link_path
```